Terminal.Gui/Core/View.cs

```diff
@@ -190,6 +190,15 @@ internal enum Direction {
 		// to make the same mistakes our users make when they poke at the Subviews.
 		internal IList<View> InternalSubviews => subviews ?? empty;
 
+		// This is null, and allocated on demand.
+		List<View> tabIndex;
```

Comment:
I read this but I ended up taking the suggestion of @migueldeicaza (https://github.com/migueldeicaza/gui.cs/issues/538#issuecomment-639240174) because it is easier to maintain.